README.md

```diff
@@ -19,12 +19,148 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Tutorial
+
+Some quick instructions for using our template are below.
+We also have a pedagogical [tutorial](./tutorial.md) that has been given in a couple of workshops geared towards researchers at UCL.
+
+## Using our template
+
+If you have [uv] installed, you can use our template with the following one-liner:
+
+```sh
+uvx cookiecutter gh:ucl-arc/python-tooling --checkout latest
+```
+
+You can also [install cookiecutter].
+Do this if you don't use [uv], or if you're likely to want to use cookiecutter again.
```

Comment:
This says "you can do this if you don't use uv", but then the command below uses `uv`. Perhaps just link to https://cookiecutter.readthedocs.io/en/stable/README.html#installation instead of offering extra commands?

Reply:
Maybe we could change

> [install cookiecutter]

to

> [install cookiecutter] with pipx or pip

to make it clearer that this is distinct from the `uv` installation instructions and maybe also change

> Here's the way we install it:

to 

> To install cookiecutter using uv run

Reply:
How about just removing the whole thing and directing the user to install cookiecutter following cookiecutter's instructions?

https://github.com/UCL-ARC/python-tooling/pull/481/commits/6816c2e03d12b495c8c5410b12d4d2fa560beb67

Then it becomes a bit simpler.